### DISCUSSION TITLE
How usable is the C++ interface?

### DISCUSSION THREAD

**Discussion by GregTheMadMonk** [EnzymeAD/Enzyme#1807] [Q&A] [answered]
Hello! I saw in the issues and in the PRs that Enzyme apparently has a C++ interface available via `#include <enzyme/enzyme>`, but couldn't find any docs on how to use it and what features it has. I see from the PR #1789 that it is buggy, but how usable will it be with this patch merged, and where do I find any info on how to use it?

**Comment by samuelpmish** [comment, marked as answer]
Hi Greg, the new interface is still experimental, so it's not quite ready to be used in anger yet. Once it is, I'll update the enzyme website and try to remember to ping you on this discussion.

**Reply by GregTheMadMonk** [reply]
Thank you! Is there anywhere besides #1789 and files mentioned there to look for the info on how it is turning out in the meantime?
